.travis.yml

```diff
@@ -1,32 +1,50 @@
 language: rust
+
+# hopefully prevent the cache from becoming huge - https://levans.fr/rust_travis_cache.html
+before_cache:
+  - rm -rf /home/travis/.cargo/registry
 cache:
   directories:
     - "$HOME/.cargo"
     - "$HOME/.ivy2/cache"
     - "$HOME/.sbt/boot/"
     - "$HOME/bin"
-# hopefully prevent the cache from becoming huge - https://levans.fr/rust_travis_cache.html
-before_cache:
-  - rm -rf /home/travis/.cargo/registry
-rust:
-  - 1.34.1
+
+# build matrix
+rust: 1.34.1
+matrix:
+  include:
+    - name: "Linux"
+      os: linux
+    - name: "OSX"
+      os: osx
+
 branches:
   only:
     - master
+
 before_install:
   - openssl aes-256-cbc -K $encrypted_487cf28eaba8_key -iv $encrypted_487cf28eaba8_iv
     -in tests/src/test/resources/service-keys.conf.enc -out tests/src/test/resources/service-keys.conf -d
 install:
   - "~/bin/sbt --help || curl -Ls https://git.io/sbt > ~/bin/sbt && chmod 0755 ~/bin/sbt"
+
+# run tests
```

Comment:
Shouldn't this go on the `script:` not `before_script`.

Reply:
Dunno, that's where it was before.

Reply:
"run tests" wasn't in the file before... https://github.com/IronCoreLabs/ironoxide-java/blob/master/.travis.yml

Reply:
Oh the comment itself. The comment was more to label what the script is for. I was including the before_script in the "what this section is doing".